### ISSUE TITLE
Gutenboarding: Pre-launch Editor experience - Intro Modal and Tooltips

### ISSUE THREAD

**Issue by ramonjd** [Automattic/wp-calypso#42385]
This is an investigation task, contingent on the outcome of the design decisions in pbAok1-LV-p2

Master thread: pbAok1-Wp-p2

## Questions

- [ ] Implementation: confirm that this will be part of the FSE plugin or an overlay in Calypso
- [ ] Feasibility of implementing positioned elements or tooltips (should that be included in the final decision)
- [ ] Determine whether the following are possible technically, and whether they should be within the scope of this project:
    * Upon landing in the Editor, let’s keep the Document side panel collapsed.
Let’s hide the “Tools”, “Content structure”, and “Block navigation” icons/buttons from the left side of the top bar. We’ll keep the “Add block”, “Undo”, and “Redo” buttons. 
    * Let’s replace the Help Gridicon (gridicons-help-outline with the white border) in the lower right with the simpler gridicons-help icon. 
    * I mocked up a post-Launch “Congratulations” side panel. Customer Home has a nice congratulatory post-launch banner at the top of the page, so this is a similar thing but within the Editor. How feasible would this be to implement and display after Launch once we return users to the Editor (if they Launch from there)? This is based on the page/post post-publish side panel, so hopefully nothing too complicated.

## Related links

Design discussion: pbAok1-LV-p2

Master thread: pbAok1-Wp-p2

**Comment by ramonjd** [comment]
Put on hold for now, but keeping here to track after design decisions

pbAok1-Xl-p2#comment-1954

**Comment by ramonjd** [comment]
A new design is in the pipeline. Closing this for now until we receive new instructions.
